src/components/Table/GridTable.tsx

```diff
@@ -891,6 +892,11 @@ function getIndentationCss<R extends Kinded>(
 ): Properties {
   // Look for cell-specific indent or row-specific indent (row-specific is only one the first column)
   const indent = (isGridCellContent(maybeContent) && maybeContent.indent) || (columnIndex === 0 && rowStyle?.indent);
+  if (typeof indent === "number" && style.levels !== undefined) {
```

Comment:
I included this as a reminder to future me of what style to use instead of `indent` :-D